cypress/integration/loginReview_spec.ts

```diff
@@ -0,0 +1,56 @@
+/// <reference types="cypress" />
+
+describe("usertesting 'Logga in'-page", () => {
+  it("login correct, visit'mina sidor', visit moviepage and write a review", () => {
+    const newText = "I liked this movie. It was awesome!! <3";
+    const username = Cypress.env("username");
+    const password = Cypress.env("password");
+
+    cy.visit("/login");
+
+    if (typeof password !== "string" || !password) {
+      throw new Error("Missing password value");
+    }
+    cy.request({
+      method: "POST",
+      url: "/api/login",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        userName: `${username}`,
+        userPassword: `${password}`,
+      }),
+    });
+    cy.get("#Username").type(username, { log: false });
+    cy.get("#userPassword").type(password, { log: false });
+
+    cy.get(".MuiButton-root")
+      .click()
+      .get(".tss-t0jjb1-logAlarm > .MuiButton-root")
+      .click();
+
+    cy.get('[href="/movies"] > .MuiButtonBase-root')
```

Comment:
this one gives an error 
![image](https://user-images.githubusercontent.com/77925373/168053441-98cb246e-b055-4c01-9a22-8d5c80af7c96.png)

Reply:
yeah, its a problem with changes in css/MUI- components. No biggie, I´ve fixed it in the branch fixed_stuff (gonna pullreq it later today) :)

Reply:
> 
aha it was this that you talked about 👍 okay!